kernel/kernel-impl/src/test/java/org/sakaiproject/content/impl/test/RoleAccessTest.java

```diff
@@ -66,11 +66,17 @@ public void setUp() throws IdUsedException, IdInvalidException, InconsistentExce
         session.setUserId("admin");
 
         _ss = (SiteService)getService(SiteService.class.getName());
-        Site newSite = _ss.addSite(SITE_ID, (String) null);
-        Group group = newSite.addGroup();
-        group.setTitle(".group");
-        _groupReference = group.getReference();
-        _ss.save(newSite);
+        Site newSite = null;
```

Comment:
I don't think this is the correct fix, you want the state to be clean between each run of the tests so you want a clean site. The better fix is to actually do the delete in the `tearDown()`.